### ISSUE TITLE
[BUG] Changing connection label error

### ISSUE THREAD

**Issue by thedist** [bitfocus/companion#2719]
### Is this a bug in companion itself or a module?

- [X] I believe this to be a bug in companion

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

When changing the label of an existing connection, the config tab in the Web UI displays the error `Failed to save connection config: "" is not a valid variable id` and the console has the error
```
warn UI/Handler Error in client handler 'connections:set-config': Error: "" is not a valid variable id Error: "" is not a valid variable id
    at SplitVariableId (/lib/Resources/Util.js:338:31)
    at VisitorReferencesUpdater.visitVariableName (/lib/Util/Visitors/ReferencesUpdater.js:138:14)
    at visitEventOptions (/lib/Resources/EventDefinitions.js:188:11)
    at DataImportExport.visitControlReferences (/lib/Data/ImportExport.js:1309:4)
    at DataImportExport.fixupControlReferences (/lib/Data/ImportExport.js:1326:8)
    at ControlTrigger.renameVariables (/lib/Controls/ControlTypes/Triggers/Trigger.js:617:51)
    at ControlsController.renameVariables (/lib/Controls/Controller.js:1564:12)
    at InstanceVariable.connectionLabelRename (/lib/Instance/Variable.js:254:17)
    at Instance.setInstanceLabelAndConfig (/lib/Instance/Controller.js:167:18)
    at /lib/Instance/Controller.js:580:10
```

Pressing 'save' on the config following this error the label change appears to work without issue.

This occurs on all modules that I have tested.

### Steps To Reproduce

1. Create a connection to a module, for example Generic HTTP
2. Edit the label of the connection and click save
3. An error will be displayed. You can either click Save again to save the new label, or disable/enable the connection also seems to load the label that was saved during the error

### Expected Behavior

The label should be updated without throwing an error (unless there is some sort of conflict with another label, or using characters that may not be allowed in the label).

### Environment (please complete the following information)

```markdown
- OS: Win 10
- Browser: Google Chrome 120.0.6099.225
- Companion Version: 3.3.0+6693-main-790985a1 (issue also present in 3.2.0+6647-beta-5aec8e7e)
```

**Comment by Julusian** [comment]
This also needs something like 
![image](https://github.com/bitfocus/companion/assets/1327476/1cb470c2-37a8-488c-961d-26789270c02c)
 to reproduce the issue.
This has a slightly different error, but is in the same area and the same root cause

**Comment by Julusian** [comment]
this is fixed in v3.2.1 and beta
